tests/data/test_tz.geojson

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "properties": {
+        "datetime": "2015-04-22T00:00:00+07:00"
```

Comment:
Here, I'd like to see the property changed from "datetime" to something like "timestamp" to make sure that the implementation doesn't depend on a special name.